src/main/java/teammates/common/datatransfer/FeedbackQuestionDetails.java

```diff
@@ -181,9 +176,12 @@ public String getNoResponseTextInHtml(String giverEmail, String recipientEmail,
                + Sanitizer.sanitizeForHtml(getNoResponseText(giverEmail, recipientEmail, bundle, question))
                + "</i>";
     }
-
-    public boolean shouldShowNoResponseText(String giverEmail, String recipientEmail,
-                                            FeedbackQuestionAttributes question) {
+    
+    /**
+     * returns true if 'No Response' to be displayed in the Response rows
+     * Used to construct missing response rows between the recipient and possible givers.
```

Comment:
"@return true if 'No Response' is to be displayed in the Response rows." is fine ?